README.md

```diff
@@ -79,6 +79,7 @@ Or on CodeSandeBox at the following url: <a href="https://codesandbox.io/s/mjxwl
 | heightMode           | `React.PropTypes.oneOf(['first', 'current', 'max'])`                                    | Change the height of the slides based either on the first slide, the current slide, or the maximum height of all slides. Overrides height set by `initialSlideHeight`                         |                     |
 | initialSlideHeight   | `React.PropTypes.number`                                                                | Initial height of the slides in pixels.                                                                                                                                                       | `100`               |
 | initialSlideWidth    | `React.PropTypes.number`                                                                | Initial width of the slides in pixels                                                                                                                                                         |                     |
+| keyboardControls     | `React.PropTypes.bool`                                                                  | When set to `false` will disable keyboard controls.                                                                                                                                           | `true`              |
```

Comment:
I would rename this to `disableKeyboardControls` instead. I think it makes it more clear what this prop is doing

Reply:
You'll have to switch the bool values in this case